### PR TITLE
hfss terminal renaming buf fix

### PR DESCRIPTION
**Pull request by svandenb-dev**

### CODE REVIEW

**Review by maxcapodi78 — COMMENTED** [review]
Please review this code.
You need to consider all possible changeproperty you may have in the terminal and apply here, not just the change name. Furthermore please add unit test

**Review by maxcapodi78 — APPROVED** [review]
LGTM

File: _unittest/test_20_HFSS.py [hunk 1]
```diff
@@ -496,6 +496,24 @@ def test_10_create_lumped_on_objects(self):
             "BoxLumped1", "BoxLumped2", self.aedtapp.AxisDir.XNeg, 50, "Lump2", False, True
         )
 
+    def test_10_1_create_lumped_ports_on_object_driven_termnal(self):
```
Comment (MaxJPRey) [inline _unittest/test_20_HFSS.py:499]:
```suggestion
    def test_10_1_create_lumped_ports_on_object_driven_terminal(self):
```

File: _unittest/test_20_HFSS.py [hunk 2]
```diff
@@ -496,6 +496,24 @@ def test_10_create_lumped_on_objects(self):
...
+        self.aedtapp.solution_type = "Terminal"
+        box1 = self.aedtapp.modeler.create_box([0, 0, 50], [10, 10, 5], "BoxLumped1")
+        box1.material_name = "Copper"
+        box2 = self.aedtapp.modeler.create_box([0, 0, 60], [10, 10, 5], "BoxLumped2")
+        box2.material_name = "Copper"
+        port = self.aedtapp.create_lumped_port_between_objects(
+            "BoxLumped1", "BoxLumped2", self.aedtapp.AxisDir.XNeg, 50, "Lump1xx", True, False
+        )
+        self.aedtapp.save_project()
+        self.aedtapp.boundaries.__init__()
```
Comment (MaxJPRey) [inline _unittest/test_20_HFSS.py]: If possible, I would avoid to call `__init__` directly.

**Pull request merged**